pyproject.toml

```
@@ -55,7 +55,7 @@ pyopenjtalk = {git = "https://github.com/VOICEVOX/pyopenjtalk", rev = "827a3fc5c
 semver = "^3.0.0"
 
 [tool.poetry.group.dev.dependencies]
-cython = "^0.29.24"
+cython = "^0.29.34,>=0.29.33"
```

ここ理由わからなくなるかもなのでpython3.11のため的なコメント書いとくのどうでしょう。
こんな感じ？
```suggestion
cython = "^0.29.34,>=0.29.33" # NOTE: for Python 3.11
```

---

Suggestion適用しました！